src/component/NavBar.tsx

```
@@ -0,0 +1,107 @@
+
+  return (
+    <nav className="flex items-center justify-between">
+      <HorizontalLogo className="mr-4 cursor-pointer" />
+      <div className="flex flex-1 items-center gap-4">
+        <button
+          onClick={() => {
+            /* 대시보드로 이동 */
+          }}
+          className="typography-body-s text-secondary-indigo cursor-pointer hover:underline hover:underline-offset-8"
+        >
+          대시보드
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)